scripts/faster_execution.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+# This script greatly speeds up execution of qutebrowser commands when a
+# qutebrowser instance already exists.  Call it in place of your system's
+# qutebrowser executable.  It is based on the following work:
+#
+# scripts/open_url_in_instance.sh
+
+import codecs
+import hashlib
+import json
+import os
+import os.path
+import socket
+import subprocess
+import sys
+
+version = '1.0.4'
+protocol_version = 1
+
+m = hashlib.md5()
+m.update(bytes(os.getenv('USER'), 'utf8'))
```

Comment:
Use `os.path.expanduser('~')` instead for a more cross-platform way of getting the home directory. Also, use `s.encode('utf8')` instead of `bytes(s, 'utf8')` please.

Reply:
The `getenv` call was performed in order to match the behavior of the original script.  I have changed it to `expanduser`.

Reply:
... and now I have changed it back.  I have questions.

Is the socket not generated by qutebrowser *outside* of this script according to `$USER`?  Given that the home directory does not necessarily correspond to user name, what good does it do here?  Your suggestion leaves me confused.